### PR TITLE
Logo

### PR DESCRIPTION
**Pull request by mcdonnelldean**
@mcollina 

I'll probably make this higher res tomorrow.

The doc structure is so I can add some documentation around patterns and that in some .md files. I can move back to just /img if you want but I don't think we need that many root folders

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# bloomrun
+![logo](./docs/imgs/bloomrun-banner.jpg)
```
Comment (mcollina) [inline README.md]: I would live the h1, because otherwise the markdown is not self-explanatory. I open READMEs in vim :).

**Comment by mcollina** [comment]
AWESOME logo! :)

**Comment by mcdonnelldean** [comment]
@mcollina done, also tidied link too

**Comment by mcollina** [comment]
Merge and release ;).

**Comment by mcdonnelldean** [comment]
@mcollina this is just a minor release right? 0.3.4

**Pull request merged**